storage/connect/tabmysql.cpp

```diff
@@ -732,7 +742,8 @@ int TDBMYSQL::MakeUpdate(PGLOBAL g)
   {
   char *qc, cmd[8], tab[96], end[1024];
 
-  Query = (char*)PlugSubAlloc(g, NULL, strlen(Qrystr) + 64);
+  size_t Query_sz = strlen(Qrystr) + 64;
+  Query = (char*)PlugSubAlloc(g, NULL, Query_sz);
```

Comment:
allocation failure?

Reply:
The logic here wasn't really changed - just added size variable to use it bellow. Allocation failure handled by `PlugSubAlloc()` - see https://github.com/MariaDB/server/blob/10.4/storage/connect/plugutil.cpp#L581-L587
Should we check the failure here as well?